doc/build.md

```diff
@@ -97,6 +97,9 @@ b) [RISCV GNU](https://github.com/riscv-collab/riscv-gnu-toolchain)
 c) [RISCV64 GCC](https://packages.ubuntu.com/bionic/gcc-riscv64-linux-gnu) for RISCV64 only
   - `sudo apt-get install gcc-riscv64-linux-gnu`
 
+c) [RISCV NONE](https://github.com/riscv/riscv-gnu-toolchain)
+  - `sudo pacman -Syu riscv32-elf-binutils riscv32-elf-newlib riscv64-elf-binutils riscv64-elf-gcc riscv64-elf-newlib`

```

Comment:
I think the index should be d). c) is already used.

I think the URL should be https://archlinux.org/packages/extra/x86_64/riscv64-elf-gcc/, right?

Reply:
Changed to `d` and the URL now points to upstream GCC as that is where the source comes form